purchase_delays/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module overrides the `product.supplierinfo` model.
+It splits the delay into two new delay fields, `Supplier Lead Time` and `Transport Lead Time`, to sum them into the actual `Lead Time`
```

Comment:
I would precise:
"This allow users to get the information about the transportation lead time in order to plan the transport properly when relevant."